conf/modules/eff_scheduling_rotwing.xml

```diff
@@ -35,7 +35,6 @@
     <file name="eff_scheduling_rotwing.h"/>
   </header>
   <init fun="eff_scheduling_rotwing_init()"/>
-  <periodic fun="eff_scheduling_rotwing_periodic()" freq="500.0"/>
```

Comment:
Not remove the whole line, only the `freq` part, as it should just run with the main loop frequency.